src/DynamoCore/Logging/DynamoLogger.cs

```diff
@@ -197,6 +202,13 @@ private void Log(string message, LogLevel level, bool reportModification)
                 if (debugSettings.VerboseLogging)
                     Analytics.LogPiiInfo("LogMessage-" + level.ToString(), message);
 
+                // In test mode, write the logs only to std out. 
+                if (testMode)
+                {
+                    Console.WriteLine(string.Format("{0} : {1}", DateTime.UtcNow.ToString("u"), message));
+                    return;
+                }
```

Comment:
Why not just leverage on the existing code for the console logging by setting the `level` variable to `LogLevel.Console` if `testMode` is true?

Reply:
The LogLevel.Console is configured to write to both the dynamo console and to the log file. So could't use it.

Reply:
You are indeed right. The fact that it does both things having that name is confusing :)